domains/rip.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ripples1253",
+        "email": "pearlcodes@outlook.com"
+    },
+    "record": {
+        "A": ["127.0.0.1"]
```

Comment:
Loop back IP, invalid.

Reply:
yeah, i did that since i didn't know my server's IP and i couldn't go onto cloudflare or use my terminal as i was on a school computer lmao

Reply:
i'll make a new PR in a day or so with my dedi's IP